ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-ref.yaml

```diff
@@ -0,0 +1,10 @@
+ref:
+  as: ipi-conf-gcp
+  from: upi-installer
+  commands: ipi-conf-gcp-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  documentation: |-
+    The IPI GCP configure step generates the GCP-specific install-config.yaml contents based on the cluster profile and optional input files.
```

Comment:
nit:
```
 documentation: >-
    <multiline here>
```

the current one is a little too long and hence not easy to read.
also maybe include the input and outputs i.e. what is required in SHARED and what is output in SHARED or any other inputs.

Reply:
Done.